New home page

chance-sematic opened the pull request:
Preview:

![capture1](https://github.com/sematic-ai/sematic/assets/133257643/5e8ab5a0-6c81-4fb2-b1a0-b98b0e0edd2f)

neutralino1 commented on sematic/ui/packages/common/src/pages/Home/blog.tsx:
```diff
@@ -0,0 +1,111 @@
+            </span>
+        </StyledSection>
+        <BlogWrapper >
+            <Link target={"_blank"} href="https://www.sematic.dev/blog/tuning-and-serving-flan-t5-and-gpt-j-6b-with-lora-sematic-and-gradio">
+            Tuning and serving FLAN-T5 and GPT-J 6B with LoRA, Sematic, and Gradio
+            </Link>
```
From what I see in the preview, I think these need to be a little bolder.

chance-sematic replied:
Fixed.

neutralino1 commented on sematic/ui/packages/common/src/pages/Home/blog.tsx:
```diff
@@ -0,0 +1,111 @@
+            <Link target={"_blank"} href="https://www.sematic.dev/blog/tuning-and-serving-flan-t5-and-gpt-j-6b-with-lora-sematic-and-gradio">
+            Tuning and serving FLAN-T5 and GPT-J 6B with LoRA, Sematic, and Gradio
+            </Link>
+            <TypographyDescription>
+            Fine-tune FLAN-T5 and GPT-J with Sematic and serve them with Gradio. Read our in-depth article.
+            </TypographyDescription>
+            <Typography color={"lightGray"}>
+            Josh Bauer on July 18, 2023.
+            </Typography>
```
I'm seeing some spacing issues here on the preview. Happy to discuss it in person.

neutralino1 reviewed (changes requested):
Looks great.
I'm seeing some font weight and spacing issues in the blog area. Happy to discuss in person.

tscurtu commented on sematic/ui/packages/common/src/component/Table.tsx:
```diff
@@ -83,19 +83,20 @@ const TableDataRow = styled(TableRow)`
 
 const TableComponent = <T,>(props: TableComponentProps<T>) => {
-    const { table, stickyHeader = true, getRowLink, className } = props;
+    const { table, headerless= false, stickyHeader = true, getRowLink, className } = props;
```
```suggestion
    const { table, headerless = false, stickyHeader = true, getRowLink, className } = props;
```

tscurtu commented on sematic/ui/packages/common/src/component/menu.tsx:
```diff
@@ -95,12 +95,14 @@ const HeaderMenu = (props: HeaderMenuProps) => {
             </MuiRouterLink>
-            <MuiRouterLink href={"https://docs.sematic.dev"} variant="subtitle1" type='menu'>Docs</MuiRouterLink>
-            <MuiRouterLink href={"https://discord.gg/4KZJ6kYVax"} variant="subtitle1" type='menu'>Discord</MuiRouterLink>
+            <MuiRouterLink href={"https://docs.sematic.dev"} target={"_blank"} variant="subtitle1" type='menu'>
```
```suggestion
            <MuiRouterLink href={"https://docs.sematic.dev"} target={"_blank"} variant="subtitle1" type="menu">
```

tscurtu commented on sematic/ui/packages/common/src/component/menu.tsx:
```diff
@@ -95,12 +95,14 @@ const HeaderMenu = (props: HeaderMenuProps) => {
-            <MuiRouterLink href={"https://discord.gg/4KZJ6kYVax"} variant="subtitle1" type='menu'>Discord</MuiRouterLink>
+            <MuiRouterLink href={"https://docs.sematic.dev"} target={"_blank"} variant="subtitle1" type='menu'>
+                Docs</MuiRouterLink>
+            <MuiRouterLink href={"https://discord.gg/4KZJ6kYVax"} target={"_blank"} variant="subtitle1" type='menu'>
```
```suggestion
            <MuiRouterLink href={"https://discord.gg/4KZJ6kYVax"} target={"_blank"} variant="subtitle1" type="menu">
```

tscurtu commented on sematic/ui/packages/common/src/component/menu.tsx:
```diff
@@ -95,12 +95,14 @@ const HeaderMenu = (props: HeaderMenuProps) => {
+                Docs</MuiRouterLink>
+            <MuiRouterLink href={"https://discord.gg/4KZJ6kYVax"} target={"_blank"} variant="subtitle1" type='menu'>
+                Discord</MuiRouterLink>
             <StyledLink variant="subtitle1" type='menu' ref={contextMenuAnchor}>
```
```suggestion
            <StyledLink variant="subtitle1" type="menu" ref={contextMenuAnchor}>
```

tscurtu commented on sematic/ui/packages/common/src/pages/Home/Community.tsx:
```diff
@@ -0,0 +1,85 @@
+    return <Container>
+        <StyledHeadline>Community</StyledHeadline>
+        <CommunityLinks onRenderLinkEntry={onRenderLinkEntry} />
+        <StyledTypography >If you have any questions, email us at support@sematic.dev.</StyledTypography>
```
```suggestion
        <StyledTypography>If you have any questions, email us at support@sematic.dev.</StyledTypography>
```

tscurtu commented on sematic/ui/packages/common/src/pages/Home/blog.tsx:
```diff
@@ -0,0 +1,111 @@
+                </Link>
+            </span>
+        </StyledSection>
+        <BlogWrapper >
```
How about extracting the contents to a separate "data" file, and here keep only logic for loading and displaying this information?

This way, we would only need to update the data file instead of updating code when rolling the blog posts.

chance-sematic replied:
Good idea. Implemented as such.

tscurtu commented on sematic/ui/packages/common/src/pages/Home/index.tsx:
```diff
@@ -0,0 +1,87 @@
+export default function Home() {
+    return (
+        <Container>
+            {/* <div id={"version-check"}>v</div> */}
```
Is this needed?

chance-sematic replied:
Nope, removed. Thanks!

tscurtu commented on sematic/ui/packages/common/src/pages/Home/latestPipelines.tsx:
```diff
@@ -0,0 +1,81 @@
+                Your latest pipelines
+            </span>
+            <span>
+                <MuiRouterLink variant="subtitle1" type='menu' href={"/pipelines"}>
```
```suggestion
                <MuiRouterLink variant="subtitle1" type="menu" href={"/pipelines"}>
```

tscurtu commented on sematic/ui/packages/common/src/pages/Home/latestRuns.tsx:
```diff
@@ -0,0 +1,81 @@
+                Your latest runs
+            </span>
+            <span>
+                <MuiRouterLink variant="subtitle1" type='menu' href={"/runs"}>
```
```suggestion
                <MuiRouterLink variant="subtitle1" type="menu" href={"/runs"}>
```

tscurtu reviewed (approved):
Thank you! :shipit:

chance-sematic commented:
The following were addressed in the new iteration:

1. removed the fixed height for each entry of the blog section
2. Adjusted spacing of the text for each entry of the blog section 
3. Removed the double border around the top part of the home page.
4. Made email address as a link on the home page.
5. Made the blog title in bold font.
